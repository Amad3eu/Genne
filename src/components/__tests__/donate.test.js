import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';

import Donate from '../donate';

describe('Donate', () => {
  it('renders correctly', () => {
    const component = shallow(<Donate />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
