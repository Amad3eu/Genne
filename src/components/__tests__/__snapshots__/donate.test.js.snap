// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Donate renders correctly 1`] = `
<Fragment>
  <div
    className="text-center text-4xl my-2"
  >
    Support 
    <span
      aria-label="praying hand emoji"
      role="img"
    >
      🙏
    </span>
  </div>
  <div
    className="flex flex-col sm:flex-row items-start justify-between"
  >
    <div
      className="w-full sm:w-2/3"
    >
      <div
        className="text-2xl mb-2"
      >
        Are you using the tool and happy with it to create your GitHub Profile?
      </div>
      <div
        className="text-lg"
      >
        Your kind support keeps open-source tools like this free for others.
      </div>
      <div
        className="mt-4"
      >
        <a
          className="flex items-center justify-start w-20"
          href="https://twitter.com/intent/tweet?text=Wow:&url=https%3A%2F%2Frahuldkjain.github.io%2Fgithub-profile-readme-generator"
        >
          <img
            alt="tweet github profile readme generator"
            className="w-20"
            src="https://img.shields.io/twitter/url?style=social&url=https%3A%2F%2Frahuldkjain.github.io%2Fgithub-profile-readme-generator"
          />
        </a>
        Let the world know how you feel using this tool. Share with others on twitter.
      </div>
    </div>
    <div
      className="w-full sm:w-1/3 flex flex-col justify-center items-center"
    >
      <span>
        Tip
        <span
          aria-label="Dollar medal"
          role="img"
        >
          💰
        </span>
      </span>
      <a
        className="flex items-center justify-evenly bg-red-500 text-white py-2 px-4 my-2"
        href="https://ko-fi.com/A0A81XXSX"
        rel="noreferrer"
        target="_blank"
      >
        <img
          alt="Buy ko-fi for rahuldkjain"
          className="w-6 h-6 mr-2"
          src="https://www.vectorlogo.zone/logos/ko-fi/ko-fi-icon.svg"
        />
        Buy me a ko-fi
      </a>
      <a
        className="flex items-center justify-evenly bg-white-500 text-white py-2 px-4 my-2 border border-solid"
        href="https://www.paypal.me/rahuldkjain/10"
        rel="noreferrer"
        target="_blank"
      >
        <img
          alt="Donate rahuldkjain via paypal"
          className="w-32 h-4"
          src="https://cdn.worldvectorlogo.com/logos/paypal-2.svg"
        />
      </a>
      <a
        className="flex items-center justify-evenly bg-orange-500 text-white py-2 px-4 my-2"
        href="https://www.buymeacoffee.com/rahuldkjain"
        rel="noreferrer"
        target="_blank"
      >
        <img
          alt="Buy rahuldkjain A Coffee"
          className="w-6 h-6 mr-2"
          src="https://www.vectorlogo.zone/logos/buymeacoffee/buymeacoffee-icon.svg"
        />
        Buy me a coffee
      </a>
    </div>
  </div>
</Fragment>
`;
